### DISCUSSION TITLE
Can Magick.Native-Q16-x64.dll be embedded into exe file?

### DISCUSSION THREAD

**Discussion by rabbitLove520** [dlemstra/Magick.NET#1290] [Help] [answered]
![image](https://user-images.githubusercontent.com/16663435/198945522-a23f991d-addc-4822-aab4-7f17f220d231.png)

**Comment by dlemstra** [comment]
Maybe? I have not tried that myself so I cannot give you an answer. Please post an answer here if you figure this out.

**Reply by rabbitLove520** [reply]
It's hard to embed, I have no idea now..

**Reply by rabbitLove520** [reply]
anyone any advice?

**Comment by rabbitLove520** [comment, marked as answer]
I have found the solution: add `/p:IncludeNativeLibrariesForSelfExtract=true`, so, for me:
```
dotnet publish -c Release -r win-x64 /p:Platform=x64 --self-contained false /p:IncludeNativeLibrariesForSelfExtract=true
```

**Reply by dlemstra** [reply]
Thanks for coming back and telling how you resolved this.
